spec/javascripts/components/feedback-spec.js

```diff
@@ -106,6 +105,20 @@ describe("Feedback component", function () {
     expect($('#something-is-wrong').find("[name=referrer]").val()).toBe("unknown");
   });
 
+  it("should append a hidden 'ga_client_id' field to the form", function() {
+    loadFeedbackComponent();
+    $('a.js-page-is-not-useful').click();
+    console.log($('#page-is-not-useful').find('#testing-feedback'))
+    expect($('#page-is-not-useful').find("[name='email_survey_signup[ga_client_id]']").val()).toBe("111111111.1111111111");
```

Comment:
You already have a test further down that tests the default value is being set correctly. This test should check what happens if the ga cookie is available. You should be able to run `window.GOVUK.cookie('_ga', 'a value here') at the start of your test so you can check this.